### PR TITLE
Update/sdk 0.25.0

### PR DESCRIPTION
**Pull request by dancespiele**
## Description

Add a description of your changes here.

## Is this PR related with an open issue?

Related to Issue #

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] Follows the code style of this project.
- [x] Tests Cover Changes
- [x] Documentation

#### Funny gif

Put a link of a funny gif inside the parenthesis-->

### CODE REVIEW

File: lib/package.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
     "react": ">=17.0.0"
   },
   "dependencies": {
-    "@nevermined-io/nevermined-sdk-js": "^0.23.5",
+    "@nevermined-io/nevermined-sdk-js": "^0.25.0-rc1",
```
Comment (aaitor) [inline lib/package.json:42]: rc2 is already available

File: lib/src/services/asset.tsx [hunk 2]
```diff
@@ -189,7 +191,7 @@ export const AssetPublishProvider = ({ children }: { children: React.ReactElemen
     method?: EncryptionMethod;
     providers?: string[];
     erc20TokenAddress?: string,
-    txParameters?: TxParameters,
+    txParameters?: string,
```
Comment (aaitor) [inline lib/src/services/asset.tsx:194]: txParameters is `TxParameters` type

**Review by aaitor — APPROVED** [review]
Looking good. There is a problem with the type of txParameters parameter but beyond that is good to go

File: lib/src/services/asset.tsx (hunk 2 above)
Comment (dancespiele) [reply lib/src/services/asset.tsx:194]: WIth the new version of SDK looks like not anymore

Comment (aaitor) [reply lib/src/services/asset.tsx:194]: you are using the `txParameter` where it should be the `appId`

**Pull request merged**